### PR TITLE
Fix some warning messages 

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

**Comment by patrickelectric** [comment]
@podhrmic @tstellanova ping just to make sure.

**Comment by tstellanova** [comment]
@patrickelectric do you get this warning message running stable rust? Or nightly?

File: build/parser.rs [hunk 1]
```diff
@@ -1037,7 +1037,7 @@ pub fn generate<R: Read, W: Write>(input: &mut R, output_rust: &mut W) {
     let rust_src = rust_tokens.into_string();
     let mut cfg = rustfmt::config::Config::default();
     cfg.set().write_mode(rustfmt::config::WriteMode::Display);
-    rustfmt::format_input(rustfmt::Input::Text(rust_src), &cfg, Some(output_rust)).unwrap();
+    let _ = rustfmt::format_input(rustfmt::Input::Text(rust_src), &cfg, Some(output_rust)).unwrap_or_else(|_| panic!("Failed to perform format."));
```
Comment (kevinmehall) [inline build/parser.rs]: This should be `.expect("Failed to perform format.")`, which would additionally log the error message from `rustfmt` in the panic message.

Comment (patrickelectric) [reply build/parser.rs]: Done.

**Comment by patrickelectric** [comment]
Any update ?

**Pull request merged**